### DISCUSSION TITLE
Challenge #3: selecting a specific image from abundance of matches

### DISCUSSION THREAD

**Discussion by CosmicKetchup** [TestPlant/sensetalk-solutions#4] [Accepted Challenges]
> One of the screens in our application has multiple copies of button and we need help with ideas on how to reliably select the right one without taking a picture of each button and associated label or text field.

#### Automation Requirement(s)
- develop an extensible solution for selecting a specific text field's corresponding button
    - must only utilize a single image-capture of the button
    - specifically will need to show validation for the **EXTR** field
        - can validate with `imageLocation` instead of interacting with button
    - should also be able to be used by other labels

#### Notes
- The necessary screenshot is already part of the suite. It can be found within the `Resources` folder.
---

<img src="https://user-images.githubusercontent.com/4438390/104521427-de85d680-55ca-11eb-8c6e-180635e00b72.png">

**Comment by james-gahan** [comment]
👍🏻

James
________________________________
